pkg/kademlia/kademlia_test.go

```diff
@@ -528,5 +528,18 @@ func newKademlia(log *zap.Logger, nodeType pb.NodeType, bootstrapNodes []pb.Node
 	}
 	transportClient := transport.NewClient(tlsOptions, rt)
 
-	return NewService(log, self, bootstrapNodes, transportClient, alpha, rt)
+	queryTimeout := 60 * time.Second
+
+	kadConfig := Config{
```

Comment:
no need to create a separate variable for config and query timeout, you can write them inline